docs/docs/getting-started/first-steps.md

```diff
@@ -179,6 +208,8 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
     constellation config fetch-measurements
     ```
 
+    TODO: Is this supported for AWS? Or do we need a hint?
```

Comment:
The upload to S3 has to happen manually for now using each AMI ID. The download should work (I think). This will be fully supported by #416